cmd/ponzu/usage.go

```diff
@@ -162,6 +162,19 @@ var usageVersion = `
 
 `
 
+var usageAdd = `
+[--cli] add, a <addon URI>
```

Comment:
All good! Thank you. I figured it was a copy/paste thing.. and I appreciate you keeping to the style of the existing commands!